pt/endpoints/powerdata/model.py

```diff
@@ -9,133 +11,113 @@
-            'history_id': history_id,
-            'address': address,
-        }
-        super(Demand, self).__init__(mother)
-        self.grid = data_struct['grid']
-        self.building = data_struct['building']
-        self.ess = data_struct['ess']
-        # pylint: disable=C0103
-        self.pv = data_struct['pv']
-        self.ev = data_struct['ev']
-        # pylint: enable=C0103
-    # fmt: on
-
 
 class ESS(PowerData):
     __tablename__ = "ess"
-    uuid = db.Column(db.String(40), db.ForeignKey("powerdata.uuid"), primary_key=True)
+    uuid = db.Column(UUID(), db.ForeignKey("powerdata.uuid"), primary_key=True)
     cluster = db.Column(db.Integer)
     power_display = db.Column(db.Float)
     __mapper_args__ = {"polymorphic_identity": "ESS"}
 
-    def __init__(self, data_struct, history_id, address):
-        mother = {
-            "uuid": data_struct["id"],
-            "field": data_struct["field"],
-            "updated_at": data_struct["updated_at"],
-            "history_id": history_id,
-            "address": address,
-        }
-        super(ESS, self).__init__(mother)
-        self.cluster = data_struct["cluster"]
-        self.power_display = data_struct["power_display"]
+    # def __init__(self, data_struct, history_id, address):
+    #     mother = {
+    #         "uuid": data_struct["id"],
+    #         "field": data_struct["field"],
+    #         "updated_at": data_struct["updated_at"],
+    #         "history_id": history_id,
+    #         "address": address,
+    #     }
+    #     super(ESS, self).__init__(mother)
+    #     self.cluster = data_struct["cluster"]
+    #     self.power_display = data_struct["power_display"]
 
 
 class EV(PowerData):
     __tablename__ = "ev"
-    uuid = db.Column(db.String(40), db.ForeignKey("powerdata.uuid"), primary_key=True)
+    uuid = db.Column(UUID(), db.ForeignKey("powerdata.uuid"), primary_key=True)
     cluster = db.Column(db.Integer)
     power_display = db.Column(db.Float)
     __mapper_args__ = {"polymorphic_identity": "EV"}
 
-    def __init__(self, data_struct, history_id, address):
-        mother = {
-            "uuid": data_struct["id"],
-            "field": data_struct["field"],
-            "updated_at": data_struct["updated_at"],
-            "history_id": history_id,
-            "address": address,
-        }
-        super(EV, self).__init__(mother)
-        self.cluster = data_struct["cluster"]
-        self.power_display = data_struct["power"]
+    # def __init__(self, data_struct, history_id, address):
+    #     mother = {
+    #         "uuid": data_struct["id"],
+    #         "field": data_struct["field"],
+    #         "updated_at": data_struct["updated_at"],
+    #         "history_id": history_id,
+    #         "address": address,
+    #     }
+    #     super(EV, self).__init__(mother)
+    #     self.cluster = data_struct["cluster"]
+    #     self.power_display = data_struct["power"]
 
```

Comment:
And again, we can remove this section instead of commenting it.

Reply:
This is not the latest version, all comments have been remove :)